src/graph.rs

```diff
@@ -1008,8 +1091,7 @@ impl StackGraph {
     /// Resolves an _unknown_ node with a "real" node.  Panics if there isn't a node in the arena
     /// with the same ID as `node`.  Returns an error is that node is not an _unknown_ node.
     pub fn resolve_unknown_node(&mut self, node: Node) -> Result<(), &Node> {
-        let id = node.id().unwrap();
-        let handle = self.node_id_handles.handle_for_id(id).unwrap();
+        let handle = self.node_for_id(node.id()).unwrap();
         let arena_node = &mut self[handle];
```

Comment:
Rust question: what StackGraph field member is `self[handle]` implicitly using? I was trying to clarify this syntax but finding it hard to find this in the Rust documentation.

Reply:
It hits the [`Index`](https://doc.rust-lang.org/std/ops/trait.Index.html) or [`IndexMut`](https://doc.rust-lang.org/std/ops/trait.IndexMut.html) trait, depending on whether it's in an immutable or mutable context.